### PR TITLE
Provide a default pressure range if a MotionEvent does not have a device

### PR DESCRIPTION
**Pull request by jason-simmons**
Fixes https://github.com/flutter/flutter/issues/27641

### CODE REVIEW

File: shell/platform/android/io/flutter/view/FlutterView.java [hunk 1]
```diff
@@ -472,10 +472,15 @@ private void addPointerForIndex(MotionEvent event, int pointerIndex, int pointer
...
-        InputDevice.MotionRange pressureRange = event.getDevice().getMotionRange(MotionEvent.AXIS_PRESSURE);
         packet.putDouble(event.getPressure(pointerIndex)); // pressure
-        packet.putDouble(pressureRange.getMin()); // pressure_min
-        packet.putDouble(pressureRange.getMax()); // pressure_max
+        if (event.getDevice() != null) {
+            InputDevice.MotionRange pressureRange = event.getDevice().getMotionRange(MotionEvent.AXIS_PRESSURE);
+            packet.putDouble(pressureRange.getMin()); // pressure_min
+            packet.putDouble(pressureRange.getMax()); // pressure_max
+        } else {
+            packet.putDouble(0.0); // pressure_min
```
Comment (Hixie) [inline shell/platform/android/io/flutter/view/FlutterView.java]: default should probably be 1.0

Comment (jason-simmons) [reply shell/platform/android/io/flutter/view/FlutterView.java]: A typical device uses pressure values ranging from 0 to 1

https://developer.android.com/reference/android/view/MotionEvent.html#getPressure(int)

**Comment by Hixie** [comment]
test?

**Comment by jason-simmons** [comment]
We don't have a test environment for Android Java code

**Pull request merged**